cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake

```diff
@@ -28,7 +28,7 @@ function(find_and_configure_cudf VERSION)
     endif()
 endfunction()
 
-set(CUDA_KAFKA_MIN_VERSION_cudf "${CUDA_KAFKA_VERSION_MAJOR}.${CUDA_KAFKA_VERSION_MINOR}")
+set(CUDA_KAFKA_MIN_VERSION_cudf "${CUDA_KAFKA_VERSION_MAJOR}.${CUDA_KAFKA_VERSION_MINOR}.00")
```

Comment:
How does someone reliably rely on a libcudf install? Should we remove the patch number from the libcudf cmake instead?

Reply:
@robertmaynard is working on answering that question in 21.08, but might have an answer for you now.

Reply:
This is to work around a [CMake bug](https://gitlab.kitware.com/cmake/cmake/-/issues/22207).

Reply:
Can we use `${CUDF_KAFKA_VERSION_PATCH}` then instead?

Reply:
it's a minimum version, so is that necessary?

Reply:
Didn't realize it's a minimum, perfect then!